code/__defines/species_languages_vr.dm

```diff
@@ -18,8 +18,9 @@
 #define LANGUAGE_MOUSE "Mouse"
 
 #define LANGUAGE_SHADEKIN "Shadekin Empathy"
+#define LANGUAGE_MARISH "Marish" // CHOMPedit
```

Comment:
Why is it not directly under the chomp languages? There's an entire CHOMPEdit language section below